File: 1069.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int i, j, n, balance, contador;
		String entrada;
		char c;
		
		n = sc.nextInt();
		sc.nextLine();
		
		for(i = 0; i < n; i++) {
			balance = 0;
			contador = 0;
			entrada = sc.nextLine();
			
			for(j = 0; j < entrada.length(); j++) {
				c = entrada.charAt(j);
				
				if(c == '<')
					balance++;
				else if(balance > 0 && c == '>') {
					balance--;
					contador++;
				}
			}
			
			System.out.println(contador);
		}
		
			
		sc.close();
	}
}

File: 1069.js
const fs = require('fs');
const data = fs.readFileSync(0, 'utf8').trim().split(/\s+/);

let index = 0;
let n = parseInt(data[index++], 10);
for (let i = 0; i < n; i++) {
    let balance = 0;
    let contador = 0;

    entrada = data[index++];
    for (let j = 0; j < entrada.length; j++) {
        if (entrada[j] === '<') {
            balance++;
        } else if (balance > 0 && entrada[j] === '>') {
            balance--;
            contador++;
        }
    }
    console.log(contador);
}